doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -112,6 +112,10 @@ nRF Desktop
 
   * Documentation for selective HID report subscription in :ref:`nrf_desktop_usb_state` using :kconfig:option:`CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION` option.
 
+* Removed:
+
+  * Configurations without bootloader. By default all boards have B0 bootloader enabled if it fits the memory.
```

Comment:
Isn't it only for the one mentioned in the nRFdesktop_readme?

Reply:
No, the change I do is about making configuration with b0 default configuration